charts/wildfly/values.schema.json

```diff
@@ -659,6 +691,15 @@
                   }
                 }
             }
+        },
+        "extraObjects": {
+          "type": "array",
+          "description": "Defines additional Kubernetes resources to be created.",
```

Comment:
Maybe we could enhance the descriptions to clarify that this is meant for resources required to successfully deploy the WildFly deployment.
Something like `Defines additional Kubernetes resources required to deploy WildFly (e.g PVC, secrets, configmaps)`